client/reducers/eventReducers.js

```diff
@@ -30,17 +30,9 @@ export const events = (state = initialState.events, action) => {
         }
       );
 
-
     case CREATE_EVENT: {
-      const createdEvent = {
-        eventsList: {
-          edges: state.events.eventsList
-            ? state.events.eventsList.edges.concat({ node: action.payload.createEvent.newEvent })
-            : { node: action.payload.createEvent.newEvent },
-          pageInfo: state.events.eventsList.pageInfo,
-        },
-      };
-      return Object.assign({}, state, { events: createdEvent });
+      const newEvents = { node: action.payload.createEvent.newEvent };
```

Comment:
Just Curious. Object.assign or spread operator? What do you normally use for this codebase? I feel it should be `object assign` for consistency sake.

Reply:
@kenneth-kip The old code uses object.assign, but when I joined, I noticed that each time modification is made, they use spread. If you look at `LOAD_MORE_EVENTS` and `GET_EVENTS` that were recently updated, spread was used, hence my use of spread.

Reply:
@chibujax Sounds good. To avoid further confusion I think we can use spread for now coz its much cleaner and easier. Unless anyone objects.

Reply:
can we merge now? This PR is a blocker for another ticket cc @kenneth-kip @andela-sjames

Reply:
Sorry for the delay. Could not test this on my local but I will approve the PRs much earlier